### PR TITLE
[Suunto App] Improvements

### PR DESCRIPTION
**Pull request by FJBDev**
- Added the possibility to customize the downloaded file names https://sourceforge.net/p/mytourbook/feature-requests/198/

image

All the available file name components

image

- Added a workout filter end date

image

### CODE REVIEW

**Comment by wolfgang-ch** [comment]
I found a few minor UI issues which are fixed in https://github.com/mytourbook/mytourbook/commit/970ee5ed75ed9affe57d478c00463d8c31cbf1de

* Tab-container is not grabing the parent container
image

* The drak theme is truncating the last combobox
image

* No colon in labels
* Using swtDefault for 5,5 margins
* Make listbox larger that a scrollbar is not necessary

image

* Adjusted layout
image

File: bundles/net.tourbook.cloud/src/net/tourbook/cloud/messages.properties [hunk 1]
```diff
@@ -40,6 +42,29 @@ Dialog_UseTourTypeMappingModified_Title = Tour Type Mapping Usage Modified
...
+Filename_Component_Day               = Day\u0009
+Filename_Component_Day_Abbr          = DD
+Filename_Component_FitExtension      = Fit Extension
+Filename_Component_FitExtension_Abbr = Fit
+Filename_Component_Hour              = Hour
+Filename_Component_Hour_Abbr         = HH
+Filename_Component_Minute            = Minute
+Filename_Component_Minute_Abbr       = MM
+Filename_Component_Month             = Month
+Filename_Component_Month_Abbr        = MM
```
Comment (wolfgang-ch) [inline bundles/net.tourbook.cloud/src/net/tourbook/cloud/messages.properties:58]: Month (MM) and minute (MM) had the same abbreviation -> fixed in https://github.com/mytourbook/mytourbook/commit/7f6a84ccae1c85a9decd4d5edbf0cd4a97fc410f by using (mm) for minutes and (hh) for hours

Comment (FJBDev) [reply bundles/net.tourbook.cloud/src/net/tourbook/cloud/messages.properties:58]: I fixed it for the french translation too

**Comment by wolfgang-ch** [comment]
The border is painted here (the color values are for the dark theme)

image

image

**Pull request merged**